python-threatexchange/threatexchange/cli/hash_cmd.py

```diff
@@ -93,16 +93,16 @@ def _parse_input(
     def execute(self, api: ThreatExchangeAPI, dataset: Dataset) -> None:
 
         all_signal_types = dataset.load_cache(
```

Comment:
Discussed offline the real problem is this line right here - dataset.load_cache() should not happen, because we should only be using stateless class methods.